src/package_manager/apt.ts

```diff
@@ -2,14 +2,16 @@ import * as exec from "@actions/exec";
 import * as im from "@actions/exec/lib/interfaces";
 import * as utils from "../utils";
 
+const CONNEXT_APT_PACKAGE_NAME = "rti-connext-dds-5.3.1";  // RTI Connext
+
 const aptCommandLine: string[] = [
 	"DEBIAN_FRONTEND=noninteractive",
-	"RTI_NC_LICENSE_ACCEPTED=yes",
 	"apt-get",
 	"install",
 	"--no-install-recommends",
 	"--quiet",
 	"--yes",
+	"RTI_NC_LICENSE_ACCEPTED=yes",
```

Comment:
oops! this will need to go back at the beginning before `apt-get` - it's the `bash` syntax for setting an environment variable for the command, it needs to go before the actual executable (`apt-get`) gets called, otherwise it's interpreted as an argument to `apt-get`.